src/plugins/intel_gpu/src/kernel_selector/kernels/convolution/convolution_kernel_base.cpp

```diff
@@ -77,6 +77,12 @@ JitConstants ConvolutionKernelBase::GetJitConstants(const convolution_params& pa
             mem_consts.AddConstants({MakeJitConstant("BILINEAR_INTERPOLATION_PAD", params.bilinear_interpolation_pad)});
     }
 
+    if (!params.is_shape_agnostic) {
+        if (params.outputs[0].Batch().v == 1) {
+            mem_consts.AddConstant(MakeJitConstant("OUTPUT_BATCH_NUM_IS_1", 1));
```

Comment:
Can we have something generic instead of that? I think macro like
`${TENSOR_NAME}_IS_DYNAMIC` and/or `${DIM_NAME}_IS_DYNAMIC` can help to handle such cases

Reply:
This jitconstant is used to make b = 0 situation in kernel. I've changed the jitter name from "OUTPUT_BATCH_NUM_IS_1" to "SKIP_BATCH" to avoid misunderstanding.